Cython/Compiler/ExprNodes.py

```diff
@@ -9111,7 +9119,7 @@ def analyse_class_annotations(self, env):
                 DictItemNode(
                     entry.pos,
                     key=IdentifierStringNode(entry.pos, value=entry.name),
-                    value=entry.annotation.string
+                    value=entry.annotation.expr
```

Comment:
Why is the string not correct here? We are following [PEP-563](https://www.python.org/dev/peps/pep-0563/#abstract).